tests/test_git_repo_scanner.py

```diff
@@ -93,21 +93,6 @@ def test_extra_exclusions_get_added(self, mock_load: mock.MagicMock):
             ],
         )
 
-    @mock.patch("pygit2.Repository", new=mock.MagicMock())
-    @mock.patch("tartufo.config.load_config_from_path")
-    def test_extra_signatures_get_added(self, mock_load: mock.MagicMock):
-        mock_load.return_value = (
-            self.data_dir / "pyproject.toml",
-            {"exclude_signatures": ["foo", "bar"]},
-        )
-        test_scanner = scanner.GitRepoScanner(
-            self.global_options, self.git_options, str(self.data_dir)
-        )
-        test_scanner.load_repo("../tartufo")
-        self.assertEqual(
-            sorted(test_scanner.global_options.exclude_signatures), ["bar", "foo"]
-        )
-
```

Comment:
Why has this test been removed? We still need to be certain that signatures from the target repository are loaded and respected.

Reply:
We are no longer computing signatures as part of load_repo and instead it will be computed whenever excluded_findings is called.

Reply:
Are you saying that the signatures are no longer loaded from the remote repo? Because that is necessary functionality. This test is here to ensure that, when a repository is loaded, we find any signatures configured in that repository, and add them to list of signatures already configured. It doesn't matter so much _how_ that's accomplished, but it does matter that it does happen.